app/views/Project/EntryEdit/LeftPane/index.tsx

```diff
@@ -260,7 +259,7 @@ function LeftPane(props: Props) {
                         />
                     </QuickActionDropdownMenu>
                     <QuickActionLink
-                        to={lead?.url ?? lead?.attachment?.file ?? ''}
+                        to={lead?.url ?? lead?.attachment?.file?.url ?? ''}
```

Comment:
if we need to handle empty string, we must use || operator instead of ??

Reply:
Looks like  in this case, we don't want to pass undefined to "to" prop.
So, the current implementation looks good to me.